docs/recipes/static-asset-revisioning.md

```diff
@@ -0,0 +1,33 @@
+# Static asset revisioning
+
+Static asset revisioning by appending content hash to filenames. Make sure to set the files to [never expire](http://developer.yahoo.com/performance/rules.html#expires) for this to have an effect.
+
+```sh
+npm install --save-dev gulp gulp-rev gulp-rev-rewrite gulp-rev-delete-original
+```
+
+```js
+const gulp = require('gulp')
+const rev = require('gulp-rev')
+const revRewrite = require('gulp-rev-rewrite')
+const revDelOriginal = require('gulp-rev-delete-original')
+
+function revisionAssets() {
+  return gulp
+    // add assets you want to revision
+    .src(`dist/**/*.{css,js}`)
```

Comment:
Why would we generate the original files in `dist` just to remove them? We should show how to do this filtering in a pipeline instead of re-reading the files after output. Otherwise I don't think this recipe is useful.